rust/noosphere-sphere/src/context.rs

```diff
@@ -72,12 +72,35 @@ where
         })
     }
 
+    /// The same as [SphereContext::traverse_by_petname], but accepts a linear sequence of
+    /// petnames and attempts to recursively traverse through spheres using that sequence.
+    /// So, if the sequence is "bob", "cat", "gold", it will traverse to bob, then to bob's
+    /// cat, then to bob's cat's gold.
```

Comment:
It seems that the docs should describe the input sequence as `["gold", "cat", "bob"]` (aligning with `@gold.cat.bob` subtext)